projects/demo/src/modules/components/textfield/index.html

```diff
@@ -15,6 +15,7 @@
             *ngFor="let example of examples; let index = index"
             [component]="index + 1 | tuiComponent"
             [content]="index + 1 | tuiExample"
+            [fullsize]="index === 3"
```

Comment:
Maybe make it short instead? Long inputs do not look that good, there's no need for such a big number

Reply:
fixed